keras/src/dtype_policies/__init__.py

```diff
@@ -1,23 +1,96 @@
 from keras.src import backend
+from keras.src.api_export import keras_export
 from keras.src.dtype_policies import dtype_policy
 from keras.src.dtype_policies.dtype_policy import QUANTIZATION_MODES
+from keras.src.dtype_policies.dtype_policy import DTypePolicy
 from keras.src.dtype_policies.dtype_policy import FloatDTypePolicy
 from keras.src.dtype_policies.dtype_policy import QuantizedDTypePolicy
 from keras.src.dtype_policies.dtype_policy import QuantizedFloat8DTypePolicy
 
+ALL_OBJECTS = {
+    DTypePolicy,
+    FloatDTypePolicy,
+    QuantizedDTypePolicy,
+    QuantizedFloat8DTypePolicy,
+}
+ALL_OBJECTS_DICT = {cls.__name__: cls for cls in ALL_OBJECTS}
 
+
+@keras_export("keras.dtype_policies.serialize")
+def serialize(dtype_policy):
+    """Serializes `DTypePolicy` instance.
+
+    Args:
+        dtype_policy: A Keras `DTypePolicy` instance.
+
+    Returns:
+        `DTypePolicy` configuration dictionary.
+    """
+    from keras.src.saving import serialization_lib
+
+    return serialization_lib.serialize_keras_object(dtype_policy)
+
+
+@keras_export("keras.dtype_policies.deserialize")
+def deserialize(config, custom_objects=None):
+    """Deserializes a serialized `DTypePolicy` instance.
+
+    Args:
+        config: `DTypePolicy` configuration.
+        custom_objects: Optional dictionary mapping names (strings) to custom
+          objects (classes and functions) to be considered during
```

Comment:
Make sure to use 4-space indentation

Reply:
Fixed.

I've also fixed the same typo in `keras/src/losses/__init__.py` (where I copied the docstring, 😄)